instance/tasks.py

```diff
@@ -27,7 +27,7 @@
 from django.conf import settings
 
 from instance.github import get_username_list_from_team, get_pr_list_from_username
-from instance.models.instance import SingleVMOpenEdXInstance
+from instance.models.openedx_instance import SandboxInstance
```

Comment:
@bradenmacdonald Yup, I wasn't suggesting to do a full split out of it - agreed that it would be nice to have the openedx code in a dedicated app, eventually. I only meant to use a submodule instead of a long filename : )

For the fact that Django couldn't detect models in submodules at some points, I don't know - I found some references about this, but they mention difficulties splitting models.py into `models/....py` which seem to have been fixed with https://code.djangoproject.com/ticket/14007 (and is what we already do). I do not know if a sub-sub-module would cause issues; feel free to ignore if it's complicated.